bootstrap.sh

```diff
@@ -71,6 +71,12 @@ echo
 echo "Enter your user password"
 echo "This password will be used for Authelia login, administrative access and SSH login"
 read -s -p "Password: " user_password
+until [[ "${#user_password}" -lt 60 ]]; do
```

Comment:
Maybe I don't understand this logic but doesn't this mean "if it's less than 60 ask again"

Reply:
Nope, the opposite. "Until this is less than 60, ask again"

Reply:
Ah, thanks for explaining. TIL `until` in bash